packages/TelegramClient-UI.package/TCUChatWindow.class/instance/addMemberCount.st

```diff
@@ -0,0 +1,7 @@
+drawing
+addMemberCount
+
+	self selectedChat memberCount notNil ifTrue: [
+		self titleBar addMorphBack: (TextMorph new
+		contents: ('Members: ' , self selectedChat memberCount) asString;
+		lock)]
```

Comment:
I think this needs one more level of indentation

Reply:
resolved in 0d78fbf